packages/mui-system/src/createStyled.js

```diff
@@ -153,17 +153,18 @@ const muiStyledFunctionResolver = ({ styledArg, props, defaultTheme, themeId })
 export default function createStyled(input = {}) {
   const {
     themeId,
```

Comment:
I think `themeId` should be configured by a setter as well. The `$$material` should not be in `packages/mui-zero-styled/src/index.js` but in Material UI.